app/templates/events/volunteerInformation.html

```diff
@@ -0,0 +1,161 @@
+{% set title = "Volunteer Information" %}
+{% set page_title = eventData.name %}
+{% set eventPast = event.isPast %}
+{% set tabName = 'dietary' %}
+{% extends "eventNav.html" %}
+
+{% block styles %}
+  {{super()}}
+  <link rel="stylesheet" href="{{url_for('static', filename ='css/volunteerInformation.css') }}?u={{lastStaticUpdate}}">
+{% endblock %}
+
+{% block scripts %}
+  {{super()}}
+  <script type="module" src="/static/js/volunteerInformation.js"></script>
+{% endblock %}
+
+{% block app_content %}
+
+{{super()}}
+
+
+
+{% if attendedUser or waitlistUser or rsvpUser %}
+  <div class="noprint" id="volunteerInformationcheck">
+    <div class="row">
+      <div class="col">
+        <label><b>Volunteer Groups:</b></label><br>
+        {% if rsvpUser %}
+        <input class="displayCheckbox noprint" type="checkbox" name="selected_items" id="rsvpSelect" checked>
+        <label for="rsvpSelect">RSVP Information</label><br>
+        {% endif %}
+        {% if waitlistUser %}
+        <input class="displayCheckbox noprint" type="checkbox" name="selected_items" id="waitlistSelect" checked>
+        <label for="waitlistSelect">Waitlist</label><br>
+        {% endif %}
+        {% if attendedUser %}
+        <input class="displayCheckbox noprint" type="checkbox" name="selected_items" id="attendedSelect" checked>
+        <label for="attendedSelect">Attended</label><br>
+        {% endif %}
```

Comment:
It is a nitpick but why not just do elif down instead of everything having individual if statements.

Reply:
The elif won't work with our current implementation in JavaScript as if there was a person who was both an RSVP and an attended user they would only show up for one of those categories.